demos/human_pose_estimation_demo/src/human_pose_estimator.cpp

```diff
@@ -121,14 +121,18 @@ void HumanPoseEstimator::reshape(const cv::Mat& image){
 void HumanPoseEstimator::frameToBlobCurr(const cv::Mat& image) {
     CV_Assert(image.type() == CV_8UC3);
     InferenceEngine::Blob::Ptr input = requestCurr->GetBlob(network.getInputsInfo().begin()->first);
-    auto buffer = input->buffer().as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::U8>::value_type *>();
+    InferenceEngine::LockedMemory<const void> requestCurrBlobMapped =
+        InferenceEngine::as<InferenceEngine::MemoryBlob>(input)->rmap();
```

Comment:
```suggestion
    InferenceEngine::LockedMemory<void> requestCurrBlobMapped =
        InferenceEngine::as<InferenceEngine::MemoryBlob>(input)->wmap();
```
The same for `void HumanPoseEstimator::frameToBlobNext(const cv::Mat& image)`.